File: src/main/java/info/devels/jtte/core/Proxy.java
package info.devels.jtte.core;

import cpw.mods.fml.common.registry.GameRegistry;
import info.devels.jtte.entities.TileEntityBeacon;

public class Proxy {
    public void registerEntities()
    {
        GameRegistry.registerTileEntity(TileEntityBeacon.class, "jtte.TileEntityBeacon");
    }

    public void registerRenderInformation() {
    }
}

File: src/main/java/info/devels/jtte/core/Proxy.kt
package info.devels.jtte.core

import cpw.mods.fml.common.registry.GameRegistry
import info.devels.jtte.entities.TileEntityBeacon


open class Proxy {
    open fun registerEntities() {
        GameRegistry.registerTileEntity(TileEntityBeacon::class.java, "jtte.TileEntityBeacon")
    }

    open fun registerRenderInformation() {
    }
}
